plugins/topology/src/components/Topology/TopologyViewWorkloadComponent.tsx

```diff
@@ -126,8 +140,15 @@ const TopologyViewWorkloadComponent = ({
             sideBarOpen={sideBarOpen}
             minSideBarSize="400px"
           >
-            {loaded && dataModel?.nodes?.length === 0 ? (
-              <TopologyEmptyState />
+            {isDataModelEmpty ||
+            !(
+              topologyViewPermissionResult.allowed &&
+              catalogEntityPermissionResult.allowed
```

Comment:
@PatAKnight should we update documentation to inlcude that the user should also have `catalogEntityReadPermission` to view the topology view?

Reply:
Yep, that makes sense to me.

Reply:
Updated with the mention of the `catalog-entity, read` permission.